akka-docs/src/main/paradox/serialization.md

```diff
@@ -254,13 +259,13 @@ incompatibility as Java serialization does.
 
 A serialized remote message (or persistent event) consists of serializer-id, the manifest, and the binary payload.
 When deserializing it is only looking at the serializer-id to pick which `Serializer` to use for `fromBinary`.
-The message class (the bindings) is not used for deserialization. The manifest is only used within the 
+The message class (the bindings) is not used for deserialization. The manifest is only used within the
```

Comment:
switched to community intellij and seems it decided to trim spaces... let me know if i should revert such lines